test/viewing.jl

```diff
@@ -24,6 +24,35 @@
     v = view(p, b)
     @test nelements(v) == 6
     @test coordinates.(v) == V2[(0, 0), (1, 0), (2, 0), (0, 1), (1, 1), (0, 2)]
+
+    # convex polygons
+    t = Triangle(P2(5, 7), P2(10, 12), P2(15, 7))
+    q = Pentagon(P2(6, 1), P2(2, 10), P2(10, 16), P2(18, 10), P2(14, 1))
+    p = PolyArea(pointify(q), [pointify(t)])
+
+    g = CartesianGrid{T}(20, 20)
+    vt = view(g, t)
+    vq = view(g, q)
+    vp = view(g, p)
+    @test nelements(vt) == 36
+    @test nelements(vq) == 162
+    @test nelements(vp) == 178
```

Comment:
we need tests that are more meaningful. check if a specific index is inside or outside the geometry, e.g. `CartesianIndex(1, 1) \in vt`